src/Components/Blazor/WebAssembly.Authentication/test/WebAssemblyAuthenticationServiceCollectionExtensionsTests.cs

```diff
@@ -0,0 +1,256 @@
+        //    var host = builder.Build();
+
+        //    var componentFactory = new ComponentFactory();
+        //    componentFactory.InstantiateComponent(host.Services, typeof(AuthenticationManager<RemoteAuthenticationState>));
+        //}
+
+        [Fact]
+        public void ApiAuthorizationOptions_ConfigurationDefaultsGetApplied()
+        {
+            var builder = WebAssemblyHostBuilder.CreateDefault();
+            builder.Services.AddApiAuthorization();
+            var host = builder.Build();
+
+            var options = host.Services.GetRequiredService<IOptions<RemoteAuthenticationOptions<ApiAuthorizationProviderOptions>>>();
+
+            var paths = options.Value.AuthenticationPaths;
+
+            Assert.Equal("authentication/login", paths.LoginPath);
+            Assert.Equal("authentication/login-callback", paths.LoginCallbackPath);
+            Assert.Equal("authentication/login-failed", paths.LoginFailedPath);
+            Assert.Equal("authentication/register", paths.RegisterPath);
+            Assert.Equal("authentication/profile", paths.ProfilePath);
+            Assert.Equal("Identity/Account/Register", paths.RemoteRegisterPath);
+            Assert.Equal("Identity/Account/Manage", paths.RemoteProfilePath);
+            Assert.Equal("authentication/logout", paths.LogoutPath);
+            Assert.Equal("authentication/logout-callback", paths.LogoutCallbackPath);
+            Assert.Equal("authentication/logout-failed", paths.LogoutFailedPath);
+            Assert.Equal("authentication/logged-out", paths.LogoutSucceededPath);
+
+            var user = options.Value.UserOptions;
+            Assert.Equal("Microsoft.AspNetCore.Components.WebAssembly.Authentication.Tests", user.AuthenticationType);
+            Assert.Equal("scope", user.ScopeClaim);
+            Assert.Equal("scope", user.RoleClaim);
+            Assert.Equal("name", user.NameClaim);
+
+            Assert.Equal(
+                "_configuration/Microsoft.AspNetCore.Components.WebAssembly.Authentication.Tests",
+                options.Value.ProviderOptions.ConfigurationEndpoint);
+        }
+
+        [Fact]
+        public void ApiAuthorizationOptions_DefaultsCanBeOverriden()
+        {
+            var builder = WebAssemblyHostBuilder.CreateDefault();
+            builder.Services.AddApiAuthorization(options =>
+            {
+                options.AuthenticationPaths = new RemoteAuthenticationApplicationPathsOptions
+                {
+                    LoginPath = "a",
+                    LoginCallbackPath = "a",
+                    LoginFailedPath = "a",
+                    RegisterPath = "a",
+                    ProfilePath = "a",
+                    RemoteRegisterPath = "a",
+                    RemoteProfilePath = "a",
+                    LogoutPath = "a",
+                    LogoutCallbackPath = "a",
+                    LogoutFailedPath = "a",
+                    LogoutSucceededPath = "a",
```

Comment:
I'd suggest giving different values to all of these, otherwise the assertions can't tell whether the right values are being assigned to the right properties.